upcloud/kubernetes.go

```diff
@@ -17,13 +17,20 @@ const (
 	KubernetesClusterTaintEffectNoExecute        KubernetesClusterTaintEffect = "NoExecute"
 	KubernetesClusterTaintEffectNoSchedule       KubernetesClusterTaintEffect = "NoSchedule"
 	KubernetesClusterTaintEffectPreferNoSchedule KubernetesClusterTaintEffect = "PreferNoSchedule"
+
+	KubernetesNodeStateNodeStateFailed KubernetesNodeState = "failed"
```

Comment:
`NodeState` twice in the `const` name

Reply:
Good catch, thanks :+1: I'll push fixed.